Makefile

```diff
@@ -559,34 +556,34 @@ ci.e2e.build.all: ci.e2e.build.sles-15-nvidia
 ci.e2e.build.%:
 	make devkit.run WHAT="make e2e.build.$*"
 
-e2e.build.centos-7: centos7 docker.clean-latest-ami
+e2e.build.centos-7: centos7
```

Comment:
we should also add `--dry-run` flag as default to all targets that builds AMI.
the flag can be override from makefile args so that our release build AMI jobs can set it to `false`.

Reply:
Oops, I meant to do that!